src/providers/LastSearchTermsProvider.js

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { useNamespace } from '@folio/stripes/core';
+
+import { LastSearchTermsContext } from '../contexts';
+import { getItem, setItem } from '../storage';
+import { INIT_PAGE_CONFIG } from '../hooks/useInventoryBrowse';
+import { useLogout } from '../hooks';
+
+const LastSearchTermsProvider = ({ children }) => {
+  const [namespace] = useNamespace();
+
+  const resetStorageAfterLogout = () => {
```

Comment:
@Dmytro-Melnyshyn should this be wrapped in `useCallback`? I see it's being passed to `useLogout`.

Reply:
`useLogout` doesn't use it in dependencies, nothing should break